CONTRIBUTING.md

```diff
@@ -66,3 +66,8 @@ Now, you can run your tests on Heroku:
 ```shell
 heroku run tests
 ```
+
+### Modifying `datadog.sh`
+
+The wrapper that runs the agent when a dyno gets started, `datadog.sh`, gets sourced by Heroku, before running the dyno's command.
```

Comment:
This feels indirect. Maybe change to "`datadog.sh` is a wrapper that runs the Agent when a dyno is started. It is sourced before running the command in the dyno's procfile."